app/presenters/settings_presenter.rb

```diff
@@ -84,6 +84,12 @@ def main_props
         disable_reviews_email: seller.disable_reviews_email,
         show_nsfw_products: seller.show_nsfw_products?,
         seller_refund_policy:,
+        reply_to_emails: seller.products
```

Comment:
Seems weird to have so much code in the presenter here. Probably should be on the user object.